### PR TITLE
[DOC]:Upgrading magma version to 1.8.0

### PR DESCRIPTION
**Pull request by tmdzk**
Signed-off-by: tmdzk <timodzik@gmail.com>

Upgrading magma version to 1.8.0

## Summary

In documentation we're referencing a lot of times the version (that is unfortunately hardcoded) every time we get a new version we have to bump those

In the future we HAVE to create an .env file at the root of magma/ with handy variables like $STABLE_VERSION, or have CI do the changes for us

## Test Plan

Make sure doc is not broken

### CODE REVIEW

**Comment by tmdzk** [comment]
@ShubhamTatvamasi I need you for the latest helm/docker ids. Please do not merge this until we changed the helm charts vers AND we have a release candidate

File: docs/readmes/howtos/5g_nsa_support.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Under 5G NSA Support, we aim to bridge the connectivity gap between a 5G compati
 
 ## Message Addition
 
-The E-RAB Modification Indication and E-RAB Modification Confirmation messages were added to the MAGMA code-base in v1.5. Functioning of these messages are explained as below:
+The E-RAB Modification Indication and E-RAB Modification Confirmation messages were added to the MAGMA code-base in v1.8. Functioning of these messages are explained as below:
```
Comment (voisey) [inline docs/readmes/howtos/5g_nsa_support.md]: This looks like something that should be kept to v1.5.

File: docs/readmes/lte/build_install_magma_pkg_in_agw.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ hide_title: true
     cd ~/magma-packages/
     ```
 
-    You will need only the ones that say `magma_1.1.XXX` and `magma-sctpd_1.1.XXX` (for v1.1 versions)
+    You will need only the ones that say `magma_1.1.XXX` and `magma-sctpd_1.1.XXX` (for v1.8 versions)
```
Comment (voisey) [inline docs/readmes/lte/build_install_magma_pkg_in_agw.md]: The package names here are still specific to 1.1.

File: docs/readmes/lte/deploy_install.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ hide_title: true
 
 # Install Access Gateway on Ubuntu (Bare Metal)
 
-> NOTE: Debian-based AGW deployments are no longer supported as of v1.6. If you want to install to Debian, refer to [v1.5 of the documentation](https://magma.github.io/magma/versions).
+> NOTE: Debian-based AGW deployments are no longer supported as of v1.8. If you want to install to Debian, refer to [v1.5 of the documentation](https://magma.github.io/magma/versions).
```
Comment (voisey) [inline docs/readmes/lte/deploy_install.md]: This looks like it should be kept as v1.6.

**Review by voisey — CHANGES_REQUESTED** [review]

File: docs/readmes/basics/prerequisites.md [hunk 4]
```diff
@@ -165,8 +165,8 @@ To download Magma current version, or a specific release do the following
 git clone https://github.com/magma/magma.git
 cd magma
 
-# in case you want to use a specific version of Magma (for example v1.6)
-git checkout v1.6
+# in case you want to use a specific version of Magma (for example v1.8)
+git checkout v1.8
```
Comment (jheidbrink) [inline docs/readmes/basics/prerequisites.md]: I think this doesn't need updating.

Comment (jheidbrink) [reply docs/readmes/basics/prerequisites.md]: Then again, doesn't hurt probably, nevermind.

File: docs/readmes/orc8r/deploy_intro.md [hunk 5]
```diff
@@ -66,12 +66,33 @@ Values for recent Orchestrator releases are summarized below
 
 ### v1.8.0
 
+<<<<<<< HEAD
+Verified with Terraform version `0.15.0`.
+=======
 Verified with Terraform version `1.0.11`.
```
[the remark this answers is not in the document]
Comment (voisey) [reply docs/readmes/orc8r/deploy_intro.md]: Looks like a merge conflict was pushed

**Review by voisey — CHANGES_REQUESTED** [review]
To get the semantic PR title check to pass, can you please change the title of the PR from "[DOC]:" to "docs:"?

**Comment by tmdzk** [comment]
> docs/readmes/orc8r/deploy_intro.md

that's so so weird it's not showing up locally

**Review by voisey — APPROVED** [review]
The changes look good, but note that you still need to change the PR prefix.

**Comment by voisey** [comment]
Hi @tmdzk, what's the status here?

**Pull request merged**